src/main/java/org/broadinstitute/hellbender/tools/exome/sexgenotyper/SexGenotypeData.java

```diff
@@ -0,0 +1,120 @@
+package org.broadinstitute.hellbender.tools.exome.sexgenotyper;
+
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.param.ParamUtils;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.*;
+import java.util.stream.IntStream;
+
+/**
+ * A class for handling sex genotypes and their likelihoods.
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public final class SexGenotypeData {
+    /* mandatory */
+    private final String sampleName;
+    private final String sexGenotype;
+    private final boolean hasExtendedGenotypingInfo;
+
+    /* optional */
+    private final Map<String, Double> logLikelihoodMap;
```

Comment:
I am a little confused here, how can this be optional, isn't it the main component of the class?